python/ray/ml/predictors/integrations/lightgbm/lightgbm_predictor.py

```diff
@@ -49,8 +50,8 @@ def predict(
         self,
         data: DataBatchType,
         feature_columns: Optional[Union[List[str], List[int]]] = None,
-        **predict_kwargs
-    ) -> DataBatchType:
+        **predict_kwargs,
+    ) -> pd.DataFrame:
```

Comment:
Should we convert back to input type?

Reply:
This isn't being done in other trainers

Reply:
@amogkam ?

Reply:
IMO yes, we should convert back into input time. I think this shouldn't be blocking for alpha but we should get it in for beta. cc @richardliaw @ericl @maxpumperla ok to go with pandas for alpha?

Reply:
Seems ok.

Reply:
We may even want to fix the output type (to dict of arrays or something). Conversion can be confusing.

Reply:
yes, let's decide this later. `DataBatchType` seems wonky if it's just pandas and numpy, but _just_ having pandas also feels off.

That's something we should bring up and test in dog-fooding: what do users expect here?